testing/src/main/java/io/stargate/it/http/graphql/graphqlfirst/DeleteCustomConditionsTest.java

```diff
@@ -147,7 +147,7 @@ public void deleteWithGreaterThanPredicate() {
         CLIENT.executeKeyspaceQuery(KEYSPACE, "mutation { deleteFooGT(pk: 1, v: 100) }");
 
     // then
-    assertThat(JsonPath.<Boolean>read(response, "$.deleteFooGT")).isTrue();
+    assertThat(JsonPath.<Boolean>read(response, "$.deleteFooGT")).isFalse();
```

Comment:
Why did this change?

Reply:
Yeah that one is puzzling to me: `false` is the right answer, because we execute `DELETE ... IF v > 100` and the value is 100. The mutation is not applied, as evidenced by the fact that the row still exists (checked on the next line).

However the response is indeed `true` on `master`, so there must be a production bug there. I'll check.

Reply:
OK I see. In `DeleteFetcher` on `master`:
```java
boolean applied = !model.ifExists() || resultSet.one().getBoolean("[applied]");
```
This doesn't take into account `IF` conditions (the line was written at a time when `ifExists` was the only possible condition, and wasn't adapted later).

This got fixed as part of the larger refactoring in this PR.